core/src/main/java/com/netflix/conductor/core/utils/NoopLock.java

```diff
@@ -0,0 +1,22 @@
+package com.netflix.conductor.core.utils;
+
+import java.util.concurrent.TimeUnit;
+
+public class NoopLock implements Lock {
+    @Override
+    public void acquireLock(String lockId) {}
+
+    @Override
+    public Boolean acquireLock(String lockId, long timeToTry, TimeUnit unit) {
```

Comment:
Do we need an object here?  Can we use primitive so null pointer exceptions are avoided